code/modules/admin/admin.dm

```diff
@@ -474,7 +474,7 @@
 	if(message)
 		if(!check_rights(R_SERVER,0))
 			message = adminscrub(message,500)
-		to_chat(world, "<span class='adminnotice'><b>[usr.client.holder.fakekey ? "Administrator" : usr.key] Announces:</b></span>\n \t [message]")
+		to_chat(world, "<font size='15' color='blue'><b>[usr.client.holder.fakekey ? "Administrator" : usr.key] Announces:</b><font size='4' color='black'> </span>\n \t [message]")

```

Comment:
The `color` tags are outlawed. Please use spans. We're already overflowing with random bullshit which makes it harder to change all of these once we go through with a proper UI redesign. 

Look at `browserOutput.css` and `browserOutput.dm`. Both files need to have a new span added if you want to make a fresh one.

Reply:
@FlattestGuitar 
![image](https://user-images.githubusercontent.com/7415122/38644352-1dcd50e2-3dd8-11e8-9872-cd94ebc2d656.png)

Reply:
I know. It only became apparent after I actually tried to redo the spans.

You were right. ;c